vllm/envs.py

```diff
@@ -1154,6 +1156,14 @@ def get_vllm_port() -> Optional[int]:
     "VLLM_TUNED_CONFIG_FOLDER":
     lambda: os.getenv("VLLM_TUNED_CONFIG_FOLDER", None),
 
+    # Allows vllm use container tool
+    "VLLM_USE_CONTAINER_TOOL":
+    lambda: bool(int(os.getenv("VLLM_USE_CONTAINER_TOOL", "0"))),
+
+    # Allows harmony instructions to be injected on system messages
+    "VLLM_HARMONY_SYSTEM_INSTRUCTIONS":
+    lambda: bool(int(os.getenv("VLLM_HARMONY_SYSTEM_INSTRUCTIONS", "0"))),
```

Comment:
```suggestion
    "VLLM_GPT_OSS_USE_CONTAINER_TOOL":
    lambda: bool(int(os.getenv("VLLM_GPT_OSS_USE_CONTAINER_TOOL", "0"))),

    # Allows harmony instructions to be injected on system messages
    "VLLM_GPT_OSS_HARMONY_SYSTEM_INSTRUCTIONS":
    lambda: bool(int(os.getenv("VLLM_GPT_OSS_HARMONY_SYSTEM_INSTRUCTIONS", "0"))),
```
Feel that these logic are heavily hardcoded to gpt-oss. What about adding a `GPT_OSS` prefix?